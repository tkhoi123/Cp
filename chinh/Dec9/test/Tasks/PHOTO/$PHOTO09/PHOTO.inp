35
10 8 18 31 34 30 20 5 11 19 12 1 23 4 26 2 3 9 25 21 7 32 35 14 27 28 24 15 6 13 29 22 17 16 33 
8 22 18 25 10 31 30 5 11 19 12 1 23 2 20 3 9 21 4 7 15 34 32 35 14 27 24 6 29 17 26 16 28 13 33 
10 31 30 27 20 11 19 33 12 17 32 29 23 2 3 9 25 21 4 1 5 7 15 34 6 35 14 24 13 22 26 16 8 28 18 
10 8 18 31 30 20 5 11 12 1 23 2 19 3 9 25 4 7 15 34 21 32 35 14 27 24 6 13 29 22 17 26 16 28 33 
10 8 18 31 20 5 19 23 12 7 1 2 3 9 25 21 35 4 15 11 30 34 32 14 27 24 6 13 29 22 17 26 16 28 33 
