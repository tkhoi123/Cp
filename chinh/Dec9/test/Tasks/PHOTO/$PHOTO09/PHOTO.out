10 8 18 31 30 20 5 11 19 12 1 23 2 3 9 25 21 4 7 15 34 32 35 14 27 24 6 13 29 22 17 26 16 28 33 